### PR TITLE
[JBPM-9878] Missing jakarta dependency for jdk11 workitems-bpmn2 tests

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**:  [JBPM-9878](https://issues.redhat.com/browse/JBPM-9878)

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

File: jbpm-workitems/jbpm-workitems-bpmn2/pom.xml [hunk 1]
```diff
@@ -78,6 +78,12 @@
       <groupId>org.apache.ws.xmlschema</groupId>
       <artifactId>xmlschema-core</artifactId>
       <scope>runtime</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.bcel</groupId>
+          <artifactId>bcel</artifactId>
+        </exclusion>
+      </exclusions>
```
Comment (afalhambra) [inline jbpm-workitems/jbpm-workitems-bpmn2/pom.xml:86]: I tried out to run a full build, including tests on jdk11 without this exclusion and works fine. So probably this is not needed? wdyt?

Comment (gmunozfe) [reply jbpm-workitems/jbpm-workitems-bpmn2/pom.xml]: OK, removed, thanks for your review @afalhambra !

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
@mareknovotny this is ready to merge

**Comment by mareknovotny** [comment]
Are you  back porting this to 7.59.x too?

**Comment by gmunozfe** [comment]
> Are you back porting this to 7.59.x too?

@mareknovotny I've just opened https://github.com/kiegroup/jbpm/pull/2018

**Pull request merged**